experiments/emu_vs_emu_macro/experiment.py

```diff
@@ -0,0 +1,353 @@
+import time
```

Comment:
sorry, this is basically a copy of tests/fpga_system_tests/emu/test_emu.py.  code in the `experiments` folder is rougher than in the rest of the repository.  might be interesting to think about whether there is a better way to manage it